spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -23,37 +23,43 @@
 
 MicroProfile implementations must include the following specifications:
 
- - <<jakarta-cdi, Jakarta Contexts and Dependency Injection 3.0>>
- - <<jakarta-annotations, Jakarta Annotations 2.0>>
- - <<jakarta-jaxrs, Jakarta RESTful Web Services 3.0>>
- - <<jakarta-jsonb, Jakarta JSON Binding 2.0>>
- - <<jakarta-jsonp, Jakarta JSON Processing 2.0>>
+ - <<jakartaee-core-profile, Jakarta EE Core Profile 10>>
  - <<mp-config, MicroProfile Config 3.0>>
  - <<mp-fault-tolerance, MicroProfile Fault Tolerance 4.0>>
  - <<mp-health-check, MicroProfile Health 4.0>>
- - <<mp-jwt-auth, MicroProfile JWT Authentication 2.0>>
- - <<mp-metrics, MicroProfile Metrics 4.0>>
- - <<mp-open-api, MicroProfile OpenAPI 3.0>>
- - <<mp-opentracing, MicroProfile OpenTracing 3.0>>
+ - <<mp-jwt-auth, MicroProfile JWT Authentication 2.0>> (TODO - 2.1?)
+ - <<mp-metrics, MicroProfile Metrics 5.0>>
+ - <<mp-open-api, MicroProfile OpenAPI 3.1>>
  - <<mp-rest-client, MicroProfile Rest Client 3.0>>
+ - <<mp-telemetry, MicroProfile Telemetry 1.0>>
 
-A MicroProfile compatible implementation must pass *all* required TCKs provided by each MicroProfile specification.
-Passing the MicroProfile optional TCKs is not required. Passing the Jakarta specifications TCKs is optional.
+A MicroProfile compatible implementation must pass *all* required TCKs provided by each MicroProfile specification,
+including the Jakarta EE Core Profile 10 TCK.
+Passing the MicroProfile optional TCKs is not required.
 
 These APIs and versions guarantee application developers a minimum set of APIs, but they are not intended to be
 restrictive.
 
 [[javase]]
 === Java SE
 
-Libraries in use by the MicroProfile leverage new features provided in Java SE 8, therefore Java SE 7 and before are
-unsupported.
+MicroProfile APIs and specifications use Java SE 11 for both source and target compatibility. MicroProfile
```

Comment:
This is not true. Only MicroProfile Metrics 5.0 and MicroProfile Telemetry uses Java SE 11 while the others are still on Java SE 8.